pkgs/ffigen/errors.md

```diff
@@ -0,0 +1,23 @@
+# Errors in ffigen
```

Comment:
Maybe move this to a doc/ folder, then we don't have to clutter the toplevel ffigen dir if we add more docs.

Reply:
✅